### DISCUSSION TITLE
JS Version

### DISCUSSION THREAD

**Discussion by vladyxd** [SonMooSans/discord-bot-dashboard-next#39] [General]
Would it be possible to make a JS version? TS is overwhelming me. It's looking amazing but I'm not used to TS

**Comment by fuma-nama** [comment]
Not planned, you can migrate it on your own but won't be an official version.

The [very early template](https://github.com/fuma-nama/discord-bot-dashboard) does use JavaScript, however it has been stopped maintaining for a whole.

**Reply by vladyxd** [reply]
Do you have a discord if you are open to discuss?
I am so stuck on "Failed to load guild"
I know I have to add some features and to render them but I have no idea how.
Figured out I need to start the backend, tho I have no db structure

Edit: got it working!
